lib/senders/templates/post_add_two_step_authentication.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Can you add entries for these new templates to the `EMAIL_TYPES` map in `lib/metrics/amplitude.js`?